quic/s2n-quic-transport/src/space/session_context.rs

```diff
@@ -411,15 +440,26 @@ impl<'a, Config: endpoint::Config, Pub: event::ConnectionPublisher>
             datagram_limits.max_datagram_payload,
         );
 
-        // TODO: call self.dc.new_path(..)
+        let max_mtu = self.path_manager.max_mtu();
+        let application_params =
+            dc::ApplicationParams::new(max_mtu, &peer_flow_control_limits, self.limits);
+
+        let dc_manager = if let Some(dc_version) = dc_version {
```

Comment:
nit: move this into the `if let`
```suggestion
        let dc_manager = if let Some(dc_version) = dc_version {
            let application_params =
                dc::ApplicationParams::new(max_mtu, &peer_flow_control_limits, self.limits);
```